taker-frontend/src/types.ts

```diff
@@ -10,6 +10,20 @@ export interface WalletInfo {
     balance: number;
     address: string;
     last_updated_at: number;
+    transactions: Transaction[];
+}
+
+export interface Transaction {
```

Comment:
Isn't the convention for naming type declarations something like *.d.ts?

Reply:
I honestly don't know.